docs/en/04_Changelogs/4.0.0.md

```diff
@@ -23,6 +23,7 @@
  * `Upload::load` now stores assets directly without saving into a `File` dataobject.
  * Protected file storage is now a core Framework API. See [/developer_guides/files/file_security] for
    more information.
```

Comment:
Link to the TinyMCE upgrading guide here: http://archive.tinymce.com/wiki.php/Tutorial:Migration_guide_from_3.x. There's some removed and added plugins which devs might've blocked. For example, devs might have disallowed image insertion by removing `advimage`, which has now been recast as `image`.